examples/build-package/main.tf

```diff
@@ -27,6 +27,20 @@ module "package_dir" {
   source_path = "${path.module}/../fixtures/python3.8-app1"
 }
 
+# Create zip-archive of a single directory where "pip install" will also be executed (default for python runtime) and set temporary directory for pip install
+module "package_dir_pip_dir" {
+  source = "../../"
+
+  create_function = false
+
+  runtime = "python3.8"
+  source_path = [{
+    path             = "${path.module}/../fixtures/python3.8-app1"
+    pip_tmp_dir      = "${path.cwd}/../fixtures"
```

Comment:
Why `path.cwd` instead of `path.module`?

Reply:
Trying to remember now, but I believe that `path.module` has a `../` prefix in it. Which something down the line did not like.

Reply:
`path.module` seems to be a better choice here ([see docs about `path.cwd`](https://www.terraform.io/docs/language/expressions/references.html#path-cwd)).

Optionally, user can use something like - `abspath(path.module)` if absolute path is required.